src/server/events/gm/MapleCoconut.java

```diff
@@ -45,12 +45,12 @@
        private List<MapleCoconuts> coconuts = new LinkedList<MapleCoconuts>();
 
        public MapleCoconut(MapleMap map) {
-           super(1, 50);
+           super(map, 50);
            this.map = map;
        }
 
        public void startEvent() {
-           map.startEvent();
+           map.startEvent(null);
            for (int i = 0; i < 506; i++) {
```

Comment:
breaking change for MapleCoconut, but this event probably didnt work to begin with.